doc/gallery/classifying_shapes.rst

```diff
@@ -0,0 +1,435 @@
+.. note::
+    :class: sphx-glr-download-link-note
+
+    Click :ref:`here <sphx_glr_download_gallery_classifying_shapes.py>` to download the full example code
+.. rst-class:: sphx-glr-example-title
+
+.. _sphx_glr_gallery_classifying_shapes.py:
+
+
+Case study: Classification of shapes
```

Comment:
Did you generate this file manually, or was it generated by sphinx-gallery?

In the latter case, it shouldn't be committed in git (and always re-generated from sources).

Reply:
It was generated by sphinx-gallery. My bad.
At some point, I had in mind to provide users with a simplest way of building docs if they change the docstrings only (in `gtda...`). Hence, I wanted to keep everything that requires different libraries (like pandoc to convert  `glossary.tex` to `glossary.rst`) on git.